mr/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -239,9 +218,6 @@ static void overlayTableProperties(Configuration configuration, TableDesc tableD
       map.put(InputFormatConfig.SERIALIZED_TABLE, SerializationUtil.serializeToBase64(table));
     }
 
-    map.put(InputFormatConfig.FILE_IO, SerializationUtil.serializeToBase64(table.io()));
-    map.put(InputFormatConfig.LOCATION_PROVIDER, SerializationUtil.serializeToBase64(table.locationProvider()));
-    map.put(InputFormatConfig.ENCRYPTION_MANAGER, SerializationUtil.serializeToBase64(table.encryption()));
```

Comment:
What about the other stuff put into the map derived from the table, like `schema`, `location`, `fileFormat` etc?
Could we get rid of them too?

Reply:
Removed most of the objects. What remained is the schema and location.